src/main/scala/cromwell/engine/backend/jes/JesRuntimeInfo.scala

```diff
@@ -21,11 +21,11 @@ object JesRuntimeInfo {
     val runtimeAttributes: RuntimeAttributes = call.task.runtimeAttributes
 
     new Resources()
-      .setRamGb(runtimeAttributes.memoryGB)
-      .setCpu(runtimeAttributes.cpu)
-      .setPreemptible(runtimeAttributes.preemptible)
-      .setZones(runtimeAttributes.defaultZones.asJava)
-      .setDisks(runtimeAttributes.defaultDisks.asJava)
+      .setRamGb(runtimeAttributes.memoryGB.gb)
+      .setCpu(runtimeAttributes.cpu.count)
+      .setPreemptible(runtimeAttributes.preemptible.isPreemptible)
+      .setZones(runtimeAttributes.defaultZones.zones.asJava)
+      .setDisks(runtimeAttributes.defaultDisks.disks.asJava)
```

Comment:
Yeah, I'll admit that this is one of those times where I put something in largely to see what people say. It'd be good to figure out the right thing to do here as we'll start seeing it a lot more as we move to case classes all over the place.

I took a cruise around the tubes and it seems like a not uncommon pattern is just to name the wrapped value 'value' for the simple type wrappers.